client/blocks/continue-as-user/index.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function ContinueAsUser( { userName, redirectUrl } ) {
```

Comment:
This components should live in `client/blocks/login/continue-as-user.jsx`. It's not used anywhere else, unlike `VisitSite`, which is used also by `SiteSelector`.

Reply:
Done!